.swiftformat

```diff
@@ -0,0 +1,36 @@
+--exclude Sources/MongoSwift/MongoSwiftVersion.swift
+
+# unnecessary
+--disable andOperator
+
+# put #if to the first column
+--ifdef "outdent"
+
+# 4 spaces
+--indent 4
+
+# 0..<3 vs 0 ..< 3
+--ranges "no-space"
+
+# always want explicit acl
+--disable redundantExtensionACL
+
+# always want explicit self
+--self insert
+
+# don't need semicolons
+--semicolons "never"
+
+--disable trailingCommas
+
+# first element of collection on newline after opening brace/bracket
+--wrapcollections "before-first"
+
+# put first argument on its own line
+--wraparguments "before-first"
+
+# if x == 1 vs if 1 == x
+--disable yodaConditions
```

Comment:
this one seems reasonable to leave on to me, 1 == x looks kinda weird

Reply:
yodaConditions actually enforces the 1 == x, so it's being disabled here; I agree, look weird it does. It's called a "yoda" condition because if read it sounds like you're speaking like Yoda from star wars...lol

Reply:
lmao who would want that.....